pypeit/core/telluric.py

```diff
@@ -452,14 +532,37 @@ def tellfit(flux, thismask, arg_dict, init_from_last=None):
             Parameter vector for the object + telluric model.
```

Comment:
There doesn't appear to be a `theta` arg anymore?

Reply:
Indeed...not sure why it is here, actually! I have moved the description up to tellfit_chi2.